src/System.Security.Cryptography.Algorithms/src/System.Security.Cryptography.Algorithms.csproj

```diff
@@ -8,6 +8,7 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <DefineConstants>INTERNAL_ASYMMETRIC_IMPLEMENTATIONS</DefineConstants>
     <NoWarn>CA5351;$(NoWarn)</NoWarn>
+    <DefineConstants Condition="'$(TargetGroup)' == 'uap' or '$(TargetGroup)' == 'uapaot'">$(DefineConstants);uap</DefineConstants>
```

Comment:
uapaot should not be necessary here. I will stop commenting on these. 

Was there any reason why you have added these '$(TargetGroup)' == 'uapaot' conditions that do not seem to be necessary everywhere?

I would like to minimize number of places where "aot" shows up in corefx, and use it only in places where it is absolutely necessary. Folks get completely confused when they see "aot" show in places that has nothing to do with aot.